src/openklant/utils/validators.py

```diff
@@ -67,7 +67,8 @@ def __call__(self, value):
 
 
 validate_postal_code = CustomRegexValidator(
-    regex="^[1-9][0-9]{3} [A-Z]{2}$", message=_("Ongeldige postcode")
+    regex="^[1-9][0-9]{3} [A-Z]{2}$",
+    message=_("Postcode moet aan het volgende formaat voldoen: 1234 AB"),
```

Comment:
Let's make this as explicit as possible
```suggestion
    message=_("Postcode moet aan het volgende formaat voldoen: `1234 AB` (met spatie)"),
```